styles/scw_styles_indev/TitleFormat.yml

```diff
@@ -0,0 +1,4 @@
+extends: ?
```

Comment:
What `extends: ?` does?

Reply:
It means "do nothing for now" ?

Reply:
yeah, the rules in the `scw_styles_indev` folder are the ones that we need to create (placeholders) or ones that we've created but aren't working properly.